c7n/resources/rdscluster.py

```diff
@@ -4,121 +4,109 @@
 import itertools
 from concurrent.futures import as_completed
 from datetime import datetime, timedelta
-from itertools import chain
+from botocore.exceptions import ClientError
 
 from c7n.actions import BaseAction
 from c7n.filters import AgeFilter, CrossAccountAccessFilter, Filter, ValueFilter
 from c7n.filters.offhours import OffHour, OnHour
 import c7n.filters.vpc as net_filters
 from c7n.manager import resources
 from c7n.query import (
-    ConfigSource, QueryResourceManager, TypeInfo, DescribeSource, RetryPageIterator)
+    ConfigSource,
+    QueryResourceManager,
+    TypeInfo,
+    DescribeSource,
+    RetryPageIterator,
+)
 from c7n.resources import rds
 from c7n.filters.kms import KmsRelatedFilter
 from .aws import shape_validate
 from c7n.exceptions import PolicyValidationError
-from c7n.utils import (
-    type_schema, local_session, snapshot_identifier, chunks)
+from c7n.utils import type_schema, local_session, snapshot_identifier, chunks, get_retry
 
 from c7n.resources.rds import ParameterFilter
 from c7n.filters.backup import ConsecutiveAwsBackupsFilter
 
-log = logging.getLogger('custodian.rds-cluster')
+log = logging.getLogger("custodian.rds-cluster")
 
 
 class DescribeCluster(DescribeSource):
-
     def get_resources(self, ids):
-        resources = chain.from_iterable(
-            self.query.filter(
-                self.manager,
-                Filters=[
-                    {'Name': 'db-cluster-id', 'Values': ids_chunk}
-                ]
-            )
-            for ids_chunk in chunks(ids, 100)  # DescribeCluster filter length limit
+        return self.query.filter(
+            self.manager, **{"Filters": [{"Name": "db-cluster-id", "Values": ids}]}
         )
-        return list(resources)
 
     def augment(self, resources):
         for r in resources:
-            r['Tags'] = r.pop('TagList', ())
+            r["Tags"] = r.pop("TagList", ())
         return resources
 
 
 class ConfigCluster(ConfigSource):
-
     def load_resource(self, item):
         resource = super().load_resource(item)
-        resource.pop('TagList', None)  # we pull tags from supplementary config
+        resource.pop("TagList", None)  # we pull tags from supplementary config
```

Comment:
this is continuing to make large amounts of stylistic changes. we have a make lint/make format target in the makefile that should be used. editor automatic formatting should be disabled to avoid spurious changes.